config-dist.json

```diff
@@ -77,5 +78,9 @@
         "MAX_THRESHOLD_SECS": 40,
         "RESPONSE_THRESHOLD_SECS": 30,
         "COOLDOWN_SECS": 300
+    },
+    "TMDBPlugin": {
+      "TMDB_API_URL": "https://....",
+      "TMDB_API_KEY": "hunter2",
```

Comment:
JSON doesn't like this line having a `,` at the end when there are no entries following it.